src/Nordea/Components/AccordionMenu.elm

```diff
@@ -11,36 +26,58 @@ type alias Config =
     { isOpen : Bool }
 
 
-view : Config -> List a -> List (Attribute msg) -> List (Html msg) -> Html msg
-view config _ attrs children_ =
+view : Config -> List (Attribute msg) -> List (Html msg) -> Html msg
+view config attrs children_ =
     let
         openAttr =
             if config.isOpen then
-                attribute "open" "" :: attrs
+                attribute "open" ""
 
             else
-                attrs
-
-        attrs_ =
-            css
-                [ children [ typeSelector "summary" [ listStyle none, descendants [ class "accordion-closed-icon" [ display inlineBlock ], class "accordion-open-icon" [ display none ] ] ] ]
-                , withAttribute "open" [ children [ typeSelector "summary" [ descendants [ class "accordion-closed-icon" [ display none ], class "accordion-open-icon" [ display inlineBlock ] ] ] ] ]
-                ]
-                :: openAttr
+                attribute "closed" ""
```

Comment:
```suggestion
                fromUnstyled empty
```
I agree this section needs refactoring -- being my first ever contribution :sweat_smile: . I think using `closed` might give someone a false impression though such attribute exists on `Html.details` but I think all we have is `open`. Any other string would also do it but using the empty one seems more convenient?

Reply:
Good idea! Updated now